src/test/unit/parameter_group.ld

```diff
@@ -0,0 +1,10 @@
+SECTIONS {
+  .pg_registry :   SUBALIGN(4)  /* force 4-byte alignment for pg_regisry. Gcc default is 32 bytes, causing padding insertion and breaking the init structure. */
+  {
+    PROVIDE_HIDDEN (__pg_registry_start = .);
+    KEEP (*(.pg_registry))
+    KEEP (*(SORT(.pg_registry.*)))
+    PROVIDE_HIDDEN (__pg_registry_end = .);
+  }
+}
+INSERT AFTER .fini_array;
```

Comment:
ok, this doesn't work for me here.

```
/usr/lib/gcc/i686-pc-cygwin/4.9.2/../../../../i686-pc-cygwin/bin/ld: .fini_array not found for insert
```

I disabled the use of the linker script and generated a map file and sure enough there's no .fini_array for the unit test binary.

I'm using gcc 4.9.2 cygwin on my main PC right now.

i tried inserting it before `.data` but then i just get unresolved symbols

```
/cygdrive/d/My Documents/dev/embedded/cleanflight/src/test/../main/config/parameter_group.c:27: undefined reference to `__pg_registry_start'
/cygdrive/d/My Documents/dev/embedded/cleanflight/src/test/../main/config/parameter_group.c:27: undefined reference to `__pg_registry_end'
```

suggestions?